### PR TITLE
Remove use of MSBuild for Unity

### PR DESCRIPTION
**Pull request by davidkline-ms**
This change removes the MRTK's use of MSBuild for Unity and related documentation references.

### CODE REVIEW

File: Assets/MRTK/Core/Utilities/Editor/Setup/MixedRealityProjectConfigurator.cs [hunk 1]
```diff
@@ -43,7 +43,7 @@ public enum Configurations
             SinglePassInstancing = 5,
             OptimalRenderingPath = 5, // using the same value of SinglePassInstancing as a replacement
             SpatialAwarenessLayer,
-            [Obsolete("EnableMSBuildForUnity is obsolete and may no longer be used", true)]
+            [Obsolete("EnableMSBuildForUnity is obsolete is no longer honored.", true)]
```
Comment (keveleigh) [inline Assets/MRTK/Core/Utilities/Editor/Setup/MixedRealityProjectConfigurator.cs:46]:
```suggestion
            [Obsolete("EnableMSBuildForUnity is obsolete and is no longer honored.", true)]
```

File: Documentation/MRTK_Configuration_Dialog.md [hunk 2]
```diff
@@ -64,12 +64,6 @@ More information can be found on the [Resonance Audio documentation](https://res
 
 ![UWP Settings](Images/ConfigurationDialog/ConfigurationDialogUWPSettings.png)
 
-### Enable MSBuild for Unity
```
Comment (keveleigh) [inline Documentation/MRTK_Configuration_Dialog.md:67]: https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Tools/HolographicRemoting.html#legacy-xr-setup-instructions also has a lot of references to this tool. We have steps for using N4U, so I think we'll need to make those the main steps

Comment (davidkline-ms) [reply Documentation/MRTK_Configuration_Dialog.md:67]: nice catch. thanks!

Comment (davidkline-ms) [reply Documentation/MRTK_Configuration_Dialog.md:67]: i believe i have removed all of the relevant references.

File: Documentation/Tools/HolographicRemoting.md [hunk 3]
```diff
@@ -110,19 +48,18 @@ In the event that the installation of the DotNetAdapter cannot be performed via
...
+![DotNetWinRT Present](../Images/Tools/Remoting/DotNetWinRTPresent.png)
 
-You can also temporarily remove the adapter to workaround your issue via the following steps:
+### Removing HoloLens 2-specific remoting support
 
-1. In Unity, go to Window -> Package Manager and uninstall MSBuild for Unity.
-1. Search for DotNetWinRT.dll in your assets list in Unity and either delete the DLL or delete the Plugins (MRTK 2.2 or earlier) or Dependencies (MRTK 2.3 or later) folder that contains it a few levels up. That should remove these conflicting namespaces, while keeping MRTK around.
-1. (Optional) Navigate to MRTK / Providers / WindowsMixedReality / Shared / DotNetAdapter in your file explorer (not Unity's Assets view) and delete the `.bin` and `.obj` folders. This removes the local cache of NuGet restored packages for DotNetWinRT.
-1. If you run the MRTK Configurator again, make sure you don't re-enable MSBuild for Unity.
+If you're running into conflicts or other issues due to the presence of the DotNetWinRT adapter, please [reach out on one of our help resources](../WelcomeToMRTK.md#getting-help).
```
Comment (keveleigh) [inline Documentation/Tools/HolographicRemoting.md:62]: Should we add a section or reference to uninstalling via N4U?

File: Documentation/Tools/HolographicRemoting.md [hunk 4]
```diff
@@ -19,22 +19,9 @@ Please continue to [Legacy XR setup instructions](#legacy-xr-setup-instructions)
...
-1. Set the target platform in **File > Build Settings** to **Universal Windows Platform**
-1. If it does not automatically display, run the MRTK Configurator Utility (**Mixed Reality Toolkit > Utilities > Configure Unity Project**)
-1. Click **Apply**
-1. Open **Window > Package Manager**
-    - Ensure that the **Windows XR Plugin** is not installed
-1. Open **Edit > Project Settings > Player**
-
-    ![Windows Mixed Reality SDK](../Images/Tools/Remoting/WindowsMixedRealitySDK.png)
-
-1. Ensure that **Virtual Reality Supported** is selected and that **Windows Mixed Reality** is added to the **Virtual Reality SDKs**
```
Comment (keveleigh) [inline Documentation/Tools/HolographicRemoting.md:37]: These steps might be good to keep around, for legacy at least (ensuring Windows XR Plugin isn't installed)

**Review by keveleigh — APPROVED** [review]

**Pull request merged**